readyset-example/README.md

```diff
@@ -0,0 +1,123 @@
+Readyset Example
+===============
+
+## Overview
+
+This example demonstrates the [Kurtosis Readyset package](https://github.com/kurtosis-tech/readyset-package) in action. We have automated the Readyset's [quickstart](https://docs.readyset.io/guides/intro/quickstart/) and showcase how Kurtosis can be useful, especially for integration and end-to-end testing use cases.
+
+In this example, we use Kurtosis to set up isolated and repeatable environment that contains three services: Postgres, Readyset, and Benchmark. The package does the following:
+
+1. It first spins up a Postgres database with initial seed data, which can be found in the `seed` folder.
+2. It then spins up a Readyset service and waits until it finishes snapshotting all the tables found in the Postgres database.
+3. It spins up a Benchmark service, installs necessary dependencies, and executes the Python script available in the Quickstart documentation against the Postgres database.
+4. We create a cached query in Readyset and execute the Python script again.
+5. Finally, we return the performance benchmark output for Postgres and Readyset.
+
+Environments defined in your GitHub repository, such as this one, can run on your local laptop or CI, or in the cloud ephemerally using Kurtosis. If you're interested in our cloud offering and want to learn how to deploy and run services with ReadySet on the cloud, please reach out to us or fill out the [form](https://mp2k8nqxxgj.typeform.com/to/U1HcXT1H).
+
+## Running Kurtosis Package
+
+To run the example, first ensure that you have the latest version of the Kurtosis CLI installed. You can do this by following the installation guide and then running the command:
+
+```shell
+kurtosis run --enclave readyset-perf github.com/kurtosis-tech/awesome-kurtosis/readyset-example
+```
+
+<details>
+<summary> <b>Click here to see expected output after the package is ran successfully </b></summary>
+
+```shell
+Starlark code successfully run. Output was:
+{
+	"postgres_output": "
+Result:
+['count']
+['2418']
+
+Query latencies (in milliseconds):
+['43.97', '30.72', '27.59', '26.86', '27.73', '28.66', '29.91', '30.18', '27.58', '28.39', '27.90', '27.56', '27.76', '27.79', '27.88', '28.78', '27.02', '27.14', '27.15', '28.40']
+
+Latency percentiles (in milliseconds):
+ p95: 31.38
+
+",
+	"readyset_output": "
+Result:
+['count(coalesce(`public`.`title_ratings`.`tconst`, 0))']
+['2418']
+
+Query latencies (in milliseconds):
+['46.59', '1.69', '0.31', '0.32', '0.27', '0.25', '0.26', '0.25', '0.24', '0.24', '0.25', '0.24', '0.24', '0.25', '0.24', '0.23', '0.24', '0.25', '0.25', '0.27']
+
+Latency percentiles (in milliseconds):
+ p95: 3.94
+
+"
+}
+INFO[2023-05-11T02:28:43-04:00] ==================================================== 
+INFO[2023-05-11T02:28:43-04:00] ||          Created enclave: quiet-trail          || 
```

Comment:
shouldn't the enclave name be `readyset-perf`? because you added the name in the above command